From b9c1f35cb08d49f86a4bdae98a19f91c5b2a63d7 Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Sat, 31 Oct 2020 19:12:09 +0100
Subject: [PATCH 4/4] Solus tweaks to default taskmanager

- set Souls-sc, preffered FM and Browser as default
- set grouped icons to show tooltips
---
 applets/taskmanager/package/contents/config/main.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/applets/taskmanager/package/contents/config/main.xml b/applets/taskmanager/package/contents/config/main.xml
index d4c7998..dae99b2 100644
--- a/applets/taskmanager/package/contents/config/main.xml
+++ b/applets/taskmanager/package/contents/config/main.xml
@@ -28,7 +28,7 @@
     </entry>
     <entry name="groupedTaskVisualization" type="Enum">
       <label>What happens when clicking on a grouped task: 0 = cycle through grouped tasks, 1 = try to show tooltips, 2 = try to show present Windows effect, 3 = show textual list (AKA group dialog)</label>
-      <default>0</default>
+      <default>1</default>
     </entry>
     <entry name="groupPopups" type="Bool">
       <label>Whether groups are to be reduced to a single task button and expand into a popup or task buttons are grouped on the widget itself.</label>
@@ -77,7 +77,7 @@
     </entry>
     <entry name="launchers" type="StringList">
       <label>The list of launcher tasks on the widget. Usually .desktop file or executable URLs. Special URLs such as preferred://browser that expand to default applications are supported.</label>
-      <default>applications:systemsettings.desktop,applications:org.kde.discover.desktop,preferred://filemanager,preferred://browser</default>
+      <default>applications:solus-sc.desktop,,preferred://filemanager,preferred://browser</default>
     </entry>
     <entry name="middleClickAction" type="Enum">
       <label>What to do on middle-mouse click on a task button.</label>
-- 
2.29.2

